<body bgcolor="#f0f0f8">
<table width="100%" cellspacing=0 cellpadding=2 border=0 summary="heading">
<tr bgcolor="#6622aa">
<td valign=bottom>&nbsp;<br>
<font color="#ffffff" face="helvetica, arial">&nbsp;<br><big><big><strong>AttributeError</strong></big></big></font></td
><td align=right valign=bottom
><font color="#ffffff" face="helvetica, arial">Python 3.6.3: /usr/bin/python3.6<br>Thu Dec  7 15:10:00 2017</font></td></tr></table>
    
<p>A problem occurred in a Python script.  Here is the sequence of
function calls leading up to the error, in the order they occurred.</p>
<table width="100%" cellspacing=0 cellpadding=0 border=0>
<tr><td bgcolor="#d8bbff"><big>&nbsp;</big><a href="file:///home/velioo/training-projects/velioo-webserver/cgi-bin/test.py">/home/velioo/training-projects/velioo-webserver/cgi-bin/test.py</a> in <strong><module></strong>()</td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;&nbsp;&nbsp;6</small>&nbsp;#cgitb.enable()<br>
</tt></font></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;&nbsp;&nbsp;7</small>&nbsp;<br>
</tt></font></td></tr>
<tr><td bgcolor="#ffccee"><tt>=&gt;<small>&nbsp;&nbsp;&nbsp;&nbsp;8</small>&nbsp;form&nbsp;=&nbsp;cgi.FieldStorag()<br>
</tt></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;&nbsp;&nbsp;9</small>&nbsp;<br>
</tt></font></td></tr>
<tr><td><font color="#909090"><tt>&nbsp;&nbsp;<small>&nbsp;&nbsp;&nbsp;10</small>&nbsp;first_name&nbsp;=&nbsp;form.getvalue('first_name')<br>
</tt></font></td></tr>
<tr><td><small><font color="#909090">form <em>undefined</em>, <strong>cgi</strong>&nbsp;= &lt;module 'cgi' from '/usr/lib/python3.6/cgi.py'&gt;, cgi.FieldStorag <em>undefined</em></font></small></td></tr></table><p><strong>AttributeError</strong>: module 'cgi' has no attribute 'FieldStorag'
<br><tt><small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</small>&nbsp;</tt>args&nbsp;=
("module 'cgi' has no attribute 'FieldStorag'",)
<br><tt><small>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</small>&nbsp;</tt>with_traceback&nbsp;=
&lt;built-in method with_traceback of AttributeError object&gt;


<!-- The above is a description of an error in a Python program, formatted
     for a Web browser because the 'cgitb' module was enabled.  In case you
     are not reading this in a Web browser, here is the original traceback:

Traceback (most recent call last):
  File "cgi-bin/test.py", line 8, in &lt;module&gt;
    form = cgi.FieldStorag()
AttributeError: module 'cgi' has no attribute 'FieldStorag'

-->
